Makefile

```diff
@@ -536,6 +541,11 @@ bench: ## Run benchmarks in /benches
 	${MAYBE_ENVIRONMENT_EXEC} cargo bench --no-default-features --features "benches" ${CARGO_BENCH_FLAGS}
 	${MAYBE_ENVIRONMENT_COPY_ARTIFACTS}
 
+.PHONY: bench-remap-functions
+bench-remap-functions: ## Run remap-functions benches
+	${MAYBE_ENVIRONMENT_EXEC} CRITERION_HOME="$(CRITERION_HOME)" cargo bench --manifest-path lib/remap-functions/Cargo.toml ${CARGO_BENCH_FLAGS}
```

Comment:
I don't feel like I should need to set `CRITERION_HOME` here on this line since it is exported above, but I could not get it working otherwise.

`echo "HOME: $(CRITERION_HOME)"` was showing that it was appropriately set.